multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/steps/ExtractServicesDeploymentStep.java

```diff
@@ -0,0 +1,53 @@
+package org.cloudfoundry.multiapps.controller.process.steps;
+
+import org.cloudfoundry.multiapps.controller.client.lib.domain.CloudServiceInstanceExtended;
+import org.cloudfoundry.multiapps.controller.core.cf.CloudHandlerFactory;
+import org.cloudfoundry.multiapps.controller.core.cf.v2.ServicesCloudModelBuilder;
+import org.cloudfoundry.multiapps.controller.core.security.serialization.SecureSerialization;
+import org.cloudfoundry.multiapps.controller.process.Messages;
+import org.cloudfoundry.multiapps.controller.process.variables.Variables;
+import org.cloudfoundry.multiapps.mta.model.DeploymentDescriptor;
+import org.cloudfoundry.multiapps.mta.model.Resource;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+
+import javax.inject.Named;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Named("extractServicesDeploymentStep")
+@Scope(BeanDefinition.SCOPE_PROTOTYPE)
+public class ExtractServicesDeploymentStep extends SyncFlowableStep {
```

Comment:
This name is a bit confusing. What has 'deployment' got to do with extracting the services?